apps/aecore/lib/aecore/tx/data_tx.ex

```diff
@@ -402,12 +455,21 @@ defmodule Aecore.Tx.DataTx do
   end
 
   defp encode(tag, version, %DataTx{type: OracleResponseTx} = tx) do
+    senders =
+      for acc <- tx.senders do
+        case Identifier.encode_data(acc) do
+          {:ok, tagged_pub_key} -> tagged_pub_key
+          {:error, msg} -> {:error, msg}
+        end
+      end
+
     list = [
       tag,
       version,
-      tx.senders,
+      senders,
       tx.nonce,
       tx.payload.query_id,
+      # TODO workarounds, adjustments should be made as soon as fields types are adjusted
```

Comment:
is this todo still to be done?

Reply:
This is an old workaround , We have a case where our OracleResponseTx  have in its payload a map, while in Epoch implementation they have a binary field.